examples/integration_test.py

```diff
@@ -1140,6 +1233,21 @@ def print_usage(exitcode, reason=None):
     sys.exit(exitcode)
 
 
+def _generate_group_id():
```

Comment:
I think you can drop the underscore prefix, this is not within a class or module, nothing is being exported anyway